### ISSUE TITLE
grafana | 05-monitoring /docker-compose.yml

### ISSUE THREAD

**Issue by RogerioLS** [DataTalksClub/mlops-zoomcamp#319]
Hey guys, how are you?

I've been facing problems with grafana for three days, but at the end of the night I managed to resolve it.
I don't know if my colleagues on the course also faced a problem, but in the grafana image, it didn't work that way for me.

```sh
 graphana:
     image: grafana/grafana
```
Reading the documentation, they advised us to run it with the enterprise, so everything worked that way.
```sh
 graphana:
      image: grafana/grafana-enterprise
```

I'm not having any more problems in the future.

**Comment by alexeygrigorev** [comment]
Can you please send a PR with a fix?

**Comment by alexeygrigorev** [comment]
Closed in #328
